### DISCUSSION TITLE
I have this case when scroll down the header will be resize, any ideas code help me? i am free discussion

### DISCUSSION THREAD

**Discussion by dimasfaiz** [community/community#63370] [Programming Help]
### Select Topic Area

Question

### Body

![Screen Shot 2023-08-10 at 17 36 14](https://github.com/community/community/assets/24537417/799d8616-345c-46e9-8c01-0e7555a80926)

This case today, i want resize scroll header in shopify, Can anyone help me with the code instructions and where I can paste the code on shopify?

**Comment by arjunkumarbose** [comment]
This is how you should fix the issue:
<img width="745" alt="Screenshot 2023-08-10 at 5 39 19 PM" src="https://github.com/community/community/assets/82363264/de8ecfb2-3971-4da3-8b4f-6d7d3a1be58f">

N.B: It's just an example of the CSS code, you can adjust the CSS simply here according to height and weight you're looking for.

Mode info: https://community.shopify.com/c/shopify-design/change-nav-bar-font-size-and-spacing-for-taste-theme/td-p/1585560

**Reply by dimasfaiz** [reply]
Hello @arjunkumarbose thank you very much for this code, but i want header box resize when scroll down. Please help me thank you
